interfaces/tests/CMakeLists.txt

```diff
@@ -2,6 +2,7 @@
 
 if(${run_int_tests})
     build_test_folder(interlocked_int)
+    build_test_folder(timer_int)
     if(MSVC)
         # waiting for timer to be implemented on linux
         build_test_folder(sync_int)

```

Comment:
>sync_int [](start = 26, length = 8)

can go outside the if MSVC now, right?

Reply:
The sync one can

---
In reply to: [512804609](https://github.com/Azure/azure-c-pal/pull/48#discussion_r512804609) [](ancestors = 512804609)